sandbox/lua/encoders/schema_influx_write.lua

```diff
@@ -33,6 +33,7 @@ Config:
     name. Note that it is not possible to interpolate either the
     "Timestamp" or the "Uuid" message fields into the name, those
     values will be interpreted as referring to dynamic message fields.
+    You can disable the name prefix by setting its value to "**none**".
```

Comment:
I think it makes more sense to change the default from "name" to "" and disable the name prefix if they leave it blank. Note that this would mean you should make a note in the "Breaking Changes" section of the CHANGES.txt changelog for the 0.10.0 release.